tox.ini

```diff
@@ -30,18 +30,20 @@ commands =
 	twine check dist/*
 
 [testenv:quality]
-whitelist_externals = 
+setenv =
+	DJANGO_SETTINGS_MODULE = test_settings
```

Comment:
`pylint-django>2.4` requires `django_settings_module` path otherwise it raises error `django-not-configured`.

Reply:
Added `django-not-configured` in pylintrc disable section instead.